nestor_api/api/flask_app.py

```diff
@@ -0,0 +1,18 @@
+"""Return a initialized Flask application with the API."""
+from flask import Flask
+
+from nestor_api.api.api import create_api
+from nestor_api.api.public_routes import heartbeat
+
+
+def create_app() -> Flask:
+    """Initialize a Flask app."""
+    app = Flask(__name__)
+
+    # Note: heartbeat is exposed without authentication
+    app.register_blueprint(heartbeat.blueprint)
+
+    api = create_api()
```

Comment:
Why do we use this function rather than creating blueprint directly inside api module as it is done for heartbeat ?

Reply:
Because there is no function such as `register_blueprint` on a blueprint. It does not allow us to have easily a "root" blueprint with a `/api` prefix that would register a workflows (for instance) route under `/api` giving us a `/api/workflows`.